composer.json

```
@@ -19,7 +19,7 @@
         "drupal-composer/drupal-scaffold": "^2.0.0",
         "drupal/address": "^1.4",
         "drupal/admin_toolbar": "^1.24",
-        "drupal/auto_entitylabel": "^2.1@beta",
+        "drupal/auto_entitylabel": "3.0-beta3",
```

```suggestion
        "drupal/auto_entitylabel": "^3.0-beta3",
```
We want this to be able to float upward from here.

---

Lets leave as is. will fix on thenext  PR